source/openapi-spec.yml

```diff
@@ -0,0 +1,328 @@
+          award_year:
+            type: string
+            example: '1992'
+            description: The year the award was made
+          place_of_study:
+            type: string
+            example: University of Huddersfield
+            description: The place of study
+          awarding_body_name:
+            type: string
+            description: 'Optional. The awarding body’s name, e.g. AQA'
+          awarding_body_country:
+            type: string
+            example: GB
+          equivalency_details:
+            type: string
+    work_experience:
+        type: object
+        properties:
+          organisation_name:
+            example: Cheadle Hulme High School
+            description: The name of the employer (company or individual)
+          start_date:
+            type: string
+            format: date
+            description: The date employment began
+          end_date:
+            type: string
+            format: date
+            description: 'The date employment finished, if appropriate'
+          role:
+            type: string
+            example: Whole School Literacy Specialist
+            description: The position held by the candidate
+          description:
+            type: string
+            example: 'I lead, develop and enhance the literacy teaching practice of others...'
+            description: A brief written description of the work involved
+    reference:
+        type: object
+        properties:
+          reference_type:
+            type: string
+            enum:
+              - academic
+              - professional
+              - school_senior_leadership
+              - character
+            description: The type of the reference
+          reason_for_character_reference:
+            type: string
+            description: 'If this is a character reference, this field will contain the reason the candidate gave for not providing one of the other types'
+          email:
+            example: julia@example.com
+            type: string
+            description: The referee’s email
+          name:
+            example: Julia Wild
+            type: string
+            description: The referee’s name
+          relationship:
+            example: Julia was my mentor at Cheadle Hulme High
+            type: string
+            description: The referee’s relationship to the candidate
+          content:
+            type: string
+            example: Boris is committed to the profession of teaching...
+            description: 'Optional. The reference content provided by the referee, once it is available'
+    withdrawal:
+      type: object
+      properties:
+        reason:
+          type: string
+          description: Optional. The candidate’s reason for withdrawing
+          example: Candidate is unwell
+        date:
+          type: string
+          format: date-time
+          description: The date on which the withdrawal was received
```

Comment:
maybe: 
`The time on which ...` or `The date and time on which ...`
to distinguish this from the others attributes having `format: date`